src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+    # outside_temperature: float
+
+    # Internal heat load - people
+    people_1: float
+    people_2: float
+    heat_loss_people_1: float
+    heat_loss_people_2: float
+    start_time_people_1: List[int]
+    end_time_people_1: List[int]
+    start_time_people_2: List[int]
+    end_time_people_2: List[int]
+
+    # Internal heat load - Lighting
+    lamps_internal: float
+    lamps_external: float
+    start_time_lamps: List[int]
+    end_time_lamps: List[int]
+    time_lamps_external: float
+    wattage_lamps: float
+
+    # Equipment heat load
+    fridge_wattage: float
+    time_fridge: float
+
+    # Infiltration loads
+    changes: float
+    volume_cold_store: float
+    energy_new_air: float
+    start_time_infiltration: List[int]
+    end_time_infiltration: List[int]
+
+    run_hours: float
+
+
+MY_PATH = "C:/Users/Ilaria/CLOVER/locations/Bahraich/inputs/simulation/"
+with open(MY_PATH + "clinic.yaml") as f:
+    data_clinic = yaml.load(f, Loader=yaml.FullLoader)
+#  print(data_clinic)
+
+
+MY_CLINIC: Clinic = Clinic(**data_clinic["clinics"][0])
+
+
+def import_weather_data(building: Clinic):
+
+    data = pd.read_csv("C:/Users/Ilaria/Desktop/weatherdata.csv")
+    temperature = data["temperature"]
+    # print(temperature)
+    return temperature.to_numpy()
+
+
+def import_weather_dataframe():
+    data = pd.read_csv("C:/Users/Ilaria/Desktop/weatherdata.csv", dtype={"local_time"})
+    return data
+
+
+temperatura = import_weather_data(building=Clinic)
+
+
+def transmission_load_walls(building: Clinic, temperature):
+    """
+    Computes the transmission load of the walls in kWh for a building.
+
+    """
+
+    return (
+        building.surface_area_walls
+        * building.u_value_walls
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_doors_windows(building: Clinic, temperature):
+    """
+    Computes the transmission load of the doors and windows in kWh for a building.
+
+    """
+
+    return (
+        building.surface_area_doors_windows
+        * building.u_value_windows_doors
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_floor(building: Clinic, temperature):
+    """
+    Computes the transmission load of the floor in kWh for a building.
+
+    """
+
+    return (
+        building.floor_area
+        * building.u_value_floor
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_roof(building: Clinic, temperature):
+    """
+    Computes the transmission load of the roof in kWh for a building.
+
+    """
+
+    return (
+        building.roof_area
+        * building.u_value_roof
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def calculate_transmission_load(building: Clinic, temperature):
+    """
+    Computes the total transmission heat load in kWh for a building.
+
```

Comment:
:memo: Maybe add some more explanation about why you're calling out to these four functions. I.E., the "total" load is the sum of the walls, floor etc.